tests/fixtures/noqa/noqa.py

```diff
@@ -774,7 +774,11 @@ def many_raises_function(parameter):  # noqa: WPS238
         raise IndexError('3')
     raise TypeError('4')
 
-
+ternary_a = 1 
+ternary_b = 2
+ternary_result = ternary_a if ... else ternary_a # noqa: WPS475
+ternary_result = ternary_b if ternary_a == ternary_b else ternary_a # noqa: WPS475
+ternary_result = ternary_a if ternary_a is not None else None # noqa: WPS475
```

Comment:
No example here is enough, we test that `# noqa` works in this test.

Reply:
Should I still have 1 test in there, or is that unnecessary?

Reply:
Just one is good :)
I had a typo: not `No`, but `One` :)